content/swagger/resources/reference_entity_records/definitions/reference_entity_records.yaml

```diff
@@ -0,0 +1,104 @@
+type: array
+items:
+  $ref: ../definitions/reference_entity_record.yaml
+example: [
+  {
+    "code": "kartell",
+    "labels": {
+      "en_US": "Kartell",
+      "fr_FR": "Kartell"
+    },
+    "main_image": "0/c/b/0/0cb0c0e115dedba676f8d1ad8343ec207ab54c7b_image.jpg",
+    "values": {
+      "description": [
+        {
+          "locale": "en_US",
+          "channel": null,
+          "data": "Kartell, the Italian furniture company that sells modern and remarkable pieces of furnitures."
+        },
+        {
+          "locale": "fr_FR",
+          "channel": null,
+          "data": "Kartell, l'éditeur de meuble italien spécialisé dans la signature de belle pièces au design contemporain."
+        }
+      ],
+      "country": [
+        {
+          "locale": null,
+          "channel": null,
+          "data": "italy"
+        }
+      ],
+      "collection_overview":[
+        {
+          "locale": null,
+          "channel": null,
+          "data": "5/1/d/8/51d81dc778ba1501a8f998f3ab5797569f3b9e25_img.png",
+          "_links": {
```

Comment:
useless _links (and it's reference-entities-media-files)

Reply:
Not really useless and we already have it in the products for media files. Can we keep it? pliiiiz :)

Reply:
I corrected the link, btw.

Reply:
My bad. You're right. I removed the link in PATCH.